### ISSUE TITLE
Solving environment: failed. ResolvePackageNot Found

### ISSUE THREAD

**Issue by robinscheurwater** [mittagessen/kraken#417]
When I try to install kraken I get the following error:

```
Solving environment: failed

ResolvePackageNotFound:
  - conda-forge::pytorch[version='>=1.7.1,<1.14']
  - imagemagick[version='>=7.1.0']
  - conda-forge::torchvision
  - coremltools[version='>=3.3']
  - pyvips
```

when entered:
```
git clone https://github.com/mittagessen/kraken.git
cd kraken
conda env create -f environment_cuda.yml
```

Please help. Thank you.

**Comment by robinscheurwater** [comment]
I will be honest to you. I don't have much experience with linux. What I've tried is to install kraken on Ubuntu 20.04 on WSL2. I also tried to install kraken by entering the same commands in Anaconda Prompt on Windows which also failed. Eventually I managed to install kraken with pip install on Ubuntu 20.04 on WSL2 so without conda. I'm now running ketos train:
![image](https://user-images.githubusercontent.com/93426626/213250180-9b7af6b3-824e-4ab0-85aa-9ee9ea9cc2af.png)
